javav2/example_code/autoscale/Readme.md

```diff
@@ -0,0 +1,64 @@
+
+### Single action
+
+The following examples use the **AutoScalingClient** object:
+
+- [Creating an Auto Scaling group](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/appsync/src/main/java/com/example/appsync/CreateApiKey.java) (CreateAutoScalingGroup command)
+- [Deleting an Auto Scaling group](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/appsync/src/main/java/com/example/appsync/DeleteApiKey.java) (DeleteAutoScalingGroup command)
+- [Getting information about an Auto Scaling group](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/appsync/src/main/java/com/example/appsync/GetDataSource.java) (DescribeAutoScalingGroups command)
+- [Removing an instance from an Auto Scaling group](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/appsync/src/main/java/com/example/appsync/ListApiKeys.java) (DetachInstances command)
+
+### Scenario
+
+- [Performing various Amazon EC2 Auto Scaling operations](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/dynamodb/src/main/java/com/example/dynamodb/Scenario.java) (Multiple commands)
+
+## Running the Amazon EC2 Auto Scaling operations Java files
+
+Some of these examples perform *destructive* operations on AWS resources, such as deleting an Auto Scaling group. **Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
+
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
+
+
+ ## Testing the Amazon EC2 Auto Scaling Java files
+
+You can test the Java code examples for Amazon EC2 Auto Scaling by running a test file named **AutoScaleTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+You can run the JUnit tests from an IDE, such as IntelliJ, or from the command line. As each test runs, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+
+	Test 3 passed
+
+**WARNING**: _Running these JUnit tests manipulates real Amazon resources and might incur charges on your account._
+
+ ### Properties file
+Before running the AWS Elastic Beanstalk JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. For example, you define a crawler name used in the tests. If you do not define all values, the JUnit tests fail.
+
+Define these values to successfully run the JUnit tests:
```

Comment:
To successfully run the JUnit tests, define the following values:

Reply:
fixed